### DISCUSSION TITLE
Data model requirements

### DISCUSSION THREAD

**Discussion by david-i-berry** [wmo-im/tt-cdm#7] [General]
In the observational context, the requirements for the data model can be illustrated by the figure below (taken from ET-DRC). An observation, measurement or sample: 
- occurs somewhere, both in space and time; 
- has a phenomenon or property that is being estimated or quantified; 
- has a result specifying the value of that property;
- may be associated with a feature of interest in addition to the location;
- has a procedure associated with placing a value on  the property, i.e. how the observation was made; 
- has an observer or host associated with the observation.

Additionally, the observation, measurement or sample may:
- have an associated quality indicator, this may be via quality flags and / or uncertainty estimates;
- have a processing history indicating what has been done to the observation, what quality control has been applied etc.
- form part of a larger collection, for example as part of a set of observations from the same observer or from part of a wider observing network.
- have associated discovery metadata allowing the observation to be discovered, accessed and retrieved.

![image](https://user-images.githubusercontent.com/29403616/178505992-cebcc661-668e-4311-89f1-0418c034fecf.png)

**Comment by david-i-berry** [comment]
When it comes to satellite data do we need to store / represent the pixel level brightness temperatures?

**Reply by peterwthorne** [reply]
Wouldn't it be sensible to limit our remit to sub-orbital data? I assume there are groups under CEOS/CGMS who have oversight here - notably WG-Climate. It would simplify our lives considerably to consider solely sub-orbital data in terms of complexity and the data volumes involved?

**Comment by david-i-berry** [comment]
See also https://github.com/ET-DRC/WMO-No.1131/blob/main/user_story/4.1-observations-data-v04.pdf for requirements.
